docs/dev/projectsetup/java_api_quickstart.zh.md

```diff
@@ -304,51 +297,46 @@ quickstart/
-small for Flink. You have to manually increase it.
-In Eclipse, choose `Run Configurations -> Arguments` and write into the `VM Arguments` box: `-Xmx800m`.
-In IntelliJ IDEA recommended way to change JVM options is from the `Help | Edit Custom VM Options` menu. See [this article](https://intellij-support.jetbrains.com/hc/en-us/articles/206544869-Configuring-JVM-options-and-platform-properties) for details.
+*请注意*：对 Flink 来说，默认的 JVM 堆内存可能太小，你应当手动增加堆内存。
+在 Eclipse中，选择 `Run Configurations -> Arguments` 并在 `VM Arguments` 对应的输入框中写入：`-Xmx800m`。
+在 IntelliJ IDEA 中，推荐从菜单 `Help | Edit Custom VM Options` 来修改 JVM 选项。有关详细信息，请参阅[此文章](https://intellij-support
+.jetbrains.com/hc/en-us/articles/206544869-Configuring-JVM-options-and-platform-properties)。
 
-### Build Project
+### 构建项目
 
-If you want to __build/package your project__, go to your project directory and
-run the '`gradle clean shadowJar`' command.
-You will __find a JAR file__ that contains your application, plus connectors and libraries
-that you may have added as dependencies to the application: `build/libs/<project-name>-<version>-all.jar`.
+如果你想要 __构建/打包项目__，请在项目目录下运行 '`gradle clean shadowJar`' 命令。
+命令执行后，你将 __找到一个 JAR 文件__，里面包含了你的应用程序，以及已作为依赖项添加到应用程序的连接器和库：`build/libs/<project-name>-<version>-all.jar`。 
 
-__Note:__ If you use a different class than *StreamingJob* as the application's main class / entry point,
-we recommend you change the `mainClassName` setting in the `build.gradle` file accordingly. That way, Flink
-can run the application from the JAR file without additionally specifying the main class.
+__注意：__ 如果你使用其他类而不是 *StreamingJob* 作为应用程序的主类/入口，
+我们建议你相应地修改 `build.gradle` 文件中的 `mainClassName` 配置。
+这样，Flink 可以从 JAR 文件运行应用程序，而无需另外指定主类。
 
-## Next Steps
+## 下一步
 
-Write your application!
+开始编写应用！
 
-If you are writing a streaming application and you are looking for inspiration what to write,
-take a look at the [Stream Processing Application Tutorial]({{ site.baseurl }}/tutorials/datastream_api.html#writing-a-flink-program).
+如果你准备编写流处理应用，正在寻找灵感来写什么，
+可以看看[流处理应用程序教程]({{ site.baseurl }}/zh/tutorials/datastream_api.html#writing-a-flink-program)
 
-If you are writing a batch processing application and you are looking for inspiration what to write,
-take a look at the [Batch Application Examples]({{ site.baseurl }}/dev/batch/examples.html).
+如果你准备编写批处理应用，正在寻找灵感来写什么，
+可以看看[批处理应用程序示例]({{ site.baseurl }}/zh/dev/batch/examples.html)
 
-For a complete overview over the APIs, have a look at the
-[DataStream API]({{ site.baseurl }}/dev/datastream_api.html) and
-[DataSet API]({{ site.baseurl }}/dev/batch/index.html) sections.
+有关 API 的完整概述，请查看
+[DataStream API]({{ site.baseurl }}/zh/dev/datastream_api.html) 和
+[DataSet API]({{ site.baseurl }}/zh/dev/batch/index.html) 章节。
 
-[Here]({{ site.baseurl }}/tutorials/local_setup.html) you can find out how to run an application outside the IDE on a local cluster.
+在[这里]({{ site.baseurl }}/zh/tutorials/local_setup.html)，你可以找到如何在 IDE 之外的本地集群中运行应用程序。
```

Comment:
Looks like this link https://flink.apache.org/zh/tutorials/local_setup.html is dead.

Reply:
I think it can be assessed from this link : [https://ci.apache.org/projects/flink/flink-docs-master/zh/tutorials/local_setup.html](https://ci.apache.org/projects/flink/flink-docs-master/zh/tutorials/local_setup.html)

_{{ site.baseurl }}/zh/tutorials/local_setup.html_ can be accessed when I build and run docs locally.

Reply:
Oh sorry, my fault.